extensions/sql-database-projects/src/test/baselines/SSDTProjectBaseline.xml

```diff
@@ -63,6 +63,7 @@
   <ItemGroup>
     <Build Include="Tables\Users.sql" />
     <Build Include="Tables\Action History.sql" />
+    <Build Include="Tables\Test%40Info.sql" />
```

Comment:
What happens if people include `Test@Info.sql`?

Reply:
it shows up in the tree and builds in both VS and ADS if it's Test@Info.sql, so I don't know why SSDT encodes it. I couldn't find where in the SSDT code it does that when I did a quick search :/